src/Model/Transaction.php

```diff
@@ -70,33 +95,66 @@ public function reset(): void
         $this->dissociated = [];
     }
 
+    /**
```

Comment:
I'm not sure what's your BC break policy and whether this method can be considered internal and therefore safe to remove? Feel free to remove if you don't think we need it, although it may be a good idea to do so in the next major version.

Reply:
This method was intended to be internal (but never has been tagged that way)  and should be marked deprecated. You did it in the doc block but deprecation should be enforced by using something like:
`@trigger_error('This method is deprecated, use one of the Transaction::insert(), Transaction::update(), Transaction::remove(), Transaction::associate(), Transaction::dissociate() methods instead.', E_USER_DEPRECATED);`

Reply:
Ok, no probs. Why muting the error with `@`, though?

Reply:
Error is silenced but still get logged in the error log.

Reply:
Sure, but that is the role of 'display_errors' php.ini directive. By placing the @ there, you're making the choice whether or not to show the error for someone else. What if they are like me, and like to see all errors on all levels? :)

It's not too important, but I suggest not muting.

Reply:
Good point, I missed your comment before merging :/